Marlin/language_hr.h

```diff
@@ -167,5 +175,154 @@
 #define MSG_DELTA_CALIBRATE_Y               "Kalibriraj Y"
 #define MSG_DELTA_CALIBRATE_Z               "Kalibriraj Z"
 #define MSG_DELTA_CALIBRATE_CENTER          "Kalibriraj Središte"
+#define MSG_INFO_MENU                       "O printeru"
+#define MSG_INFO_PRINTER_MENU               "Podaci o printeru"
+#define MSG_INFO_STATS_MENU                 "Statistika printera"
+#define MSG_INFO_BOARD_MENU                 "Podaci o elektronici"
+#define MSG_INFO_THERMISTOR_MENU            "Termistori"
+#define MSG_INFO_EXTRUDERS                  "Extruderi"
+#define MSG_INFO_BAUDRATE                   "Baud"
+#define MSG_INFO_PROTOCOL                   "Protokol"
+#define MSG_LIGHTS_ON                       "Upali osvjetljenje"
+#define MSG_LIGHTS_OFF                      "Ugasi osvjetljenje"
+#if LCD_WIDTH >= 20
+  #ifndef MSG_INFO_PRINT_COUNT
+    #define MSG_INFO_PRINT_COUNT              "Broj printova"
+  #endif
```

Comment:
You will need to remove all these `#ifndef` lines. They are not supposed to be included in translations.